src/main/java/com/exasol/projectkeeper/validators/changesfile/DependencySectionFixer.java

```diff
@@ -26,16 +26,16 @@
      * @param mavenModelReader reader for maven model
      * @param projectDirectory projects root directory
      */
-    public DependencySectionFixer(final MavenFileModelReader mavenModelReader, final Path projectDirectory) {
+    public DependencySectionFixer(final MavenFileProjectReader mavenModelReader, final Path projectDirectory) {
         this.mavenModelReader = mavenModelReader;
         this.projectDirectory = projectDirectory;
         this.currentMavenModel = parseCurrentPomFile(projectDirectory.resolve("pom.xml"));
     }
 
     private Model parseCurrentPomFile(final Path pomFile) {
         try {
-            return this.mavenModelReader.readModel(pomFile.toFile());
-        } catch (final MavenFileModelReader.ReadFailedException exception) {
+            return this.mavenModelReader.readModel(pomFile.toFile()).getModel();
```

Comment:
I expect that readModel() returns model, but we have to also call getModel() afterward. Renaming should fix it